dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,487 @@
+        * **h** - height of the image in pixels, optional (integer)
+
+##### DSA
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description           | Example   | Type      |
+|---------------|----------|-----------------------|-----------|-----------|
+| `dsarequired`      |  |  flag to indicate if DSA information should be made available          | `3`    | `'integer'` |
+| `pubrender`      |  | flag to indicate if the publisher will render the DSA Transparency info          | `0`    | `'integer'` |
+| `datatopub`      |  | ndependent of pubrender, the publisher may need the transparency data for audit purposes           | `2`    | `'integer'` |
+
+```javascript
+adUnits = [{
+    code: 'postbid_iframe',
+    mediaTypes: {
+        native: {
+            ortb: {
+                assets: [{
+                    id: 2,
+                    required: 1,
+                    title: {
+                        len: 124
+                    }
+                },
+                {
+                    id: 3,
+                    data: {
+                        type: 1,
+                        len: 50
+                    }
+                },
+                {
+                    id: 1,
+                    required: 1,
+                    img: {
+                        type: 3,
+                        w: 300,
+                        h: 300
+                    }
+                }]
+            }
+        }
+    },
+    bids: [{
+        bidder: 'exadsadserver',
+        params: {
+                zoneId: 12345,
+                fid: '829a896f011475d50da0d82cfdd1af8d9cdb07ff',
+                partner: 'ortb_2_4',
+                siteId: '123',
+                siteName: 'test.com',
+                userIp: '0.0.0.0',
+                userId: '1234',
+                impressionId: '1234',
+                native: {
+                    plcmtcnt: 4
+                },
+                dsa: {
+                    dsarequired: 3, 
+                    pubrender: 0,
+                    datatopub: 2
+                },
+                country: 'IRL',
+                keywords: 'lifestyle, humour',
+                bidfloor: 0.00000011,
+                bidfloorcur: 'EUR',
+                bcat: ['IAB25', 'IAB7-39','IAB8-18','IAB8-5','IAB9-9'],
+                badv: ['first.com', 'second.com'],                
+                endpoint: 'https://your-ad-network.com/rtb.php'
+            }
+        }]
+}];
+```
+
+# DSA Transparency
+All DSA information, returned by the ad server, can be found into the **meta** tag of the response. As:
+```
+"meta": {
+    "dsa": {
+        "behalf": "...",
+        "paid": "...",
+        "transparency": [
+            {
+                "params": [
+                    ...
+                ]
+            }
+        ],
+        "adrender": ...
+    }
+}
+```
+For more information navigate to https://docs.prebid.org/dev-docs/bidder-adaptor.html.
+
+# Tools and suggestions
```

Comment:
@giuseppe-exads I wouldn't keep this "Tools and suggestions" section. It's a bit unrelated to the purpose of this doc.

Reply:
Removed section.

Reply:
Agree. We could maybe have url that would take them to our documentation where they could find this extra info, but not more than that.